Dockerfile

```diff
@@ -0,0 +1,142 @@
+################################
+# Python dependencies builder
+#
+FROM python:2-stretch AS base
```

Comment:
as a future experiment, we may want to try one of the images linked from https://github.com/mozmeao/infra/issues/960#issuecomment-433537794, specifically `revolutionsystems/python:2.7-wee`

Reply:
We can experiment with that. I think perhaps a more simple site would be a better place to start though.